index.js

```diff
@@ -0,0 +1,93 @@
+const startButton = document.querySelector(".start-button");
+const restartButton = document.querySelector(".restart-button");
+const numberInput = document.querySelector(".number-input");
+const tryCounter = document.querySelector(".try-counter");
+const ballCounter = document.querySelector(".ball-counter");
+
+// Start 클릭 시 시작 버튼 제거 및 숫자 입력칸, 안내판, 재시작 버튼 생성
+startButton.addEventListener('click', () => {
+
+    startButton.classList.add('dissapear');
+    numberInput.classList.remove('dissapear');
+    tryCounter.classList.remove('dissapear');
+    ballCounter.classList.remove('dissapear');
+    restartButton.classList.remove('dissapear');
+    gameStart();
+})
+
+let enterCount = 10; // 시도 제한 횟수 : 10회
+
+// Restart 클릭 시 초기화
+restartButton.addEventListener('click', () => {
+
+    (0 < enterCount && enterCount <= 10)
+    ? enterCount = enterCount + (10 - enterCount)
+    : enterCount = enterCount + 10;
+
+    allNum.push(...answer);
+    gameStart();
+})
+
+// 입력창 엔터키 사용 시 답안 판별
+numberInput.addEventListener('keyup', (event) => {
+
+    let numbers = numberInput.value;
+    let strike = 0;
+    let ball = 0;
+
+    if (event.key === "Enter") {
+
+        if (enterCount !== 0) {
+
+            if (numbers.length !== 3) {
+                alert("세 자리 숫자를 입력하세요.");
+
+            } else {
+                enterCount--;
+
+                for (i=0; i<3; i++) {
```

Comment:
```suggestion
                for (let i = 0; i < 3; i++) {
```
변수를 선언하고 사용해주세요. 띄어쓰기도 체크해주세요!
* 참고 - [MDN-for](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/for)